rest-services/server/src/test/java/com/_4point/aem/docservices/rest_services/server/assembler/AssembleDocumentsTest.java

```diff
@@ -96,15 +111,17 @@ void testDoPost_HappyPath_JustForm() throws ServletException, IOException, NoSuc
 		GenerateAssemblerResultArgs generateAssemblerResultArgs = assemblePdfMock.getGenerateAssemblerResultArgs();
 		assertNotNull(generateAssemblerResultArgs.getDdx());
         AssemblerOptionsSpec assemblerOptionsSpec =
-		  generateAssemblerResultArgs.getAssemblerOptionsSpec(); assertAll(
+		  generateAssemblerResultArgs.getAssemblerOptionsSpec(); 
+        assertAll(
 		  ()->assertNotNull(assemblerOptionsSpec.getLogLevel()),
-		  ()->assertNull(assemblerOptionsSpec.isFailOnError()),
-		  ()->assertNull(assemblerOptionsSpec.isTakeOwnership()),
-		  ()->assertNull(assemblerOptionsSpec.isValidateOnly()),
-		  ()->assertNull(assemblerOptionsSpec.getDefaultStyle()),
+		  ()->assertNotNull(assemblerOptionsSpec.isFailOnError()),
+		  ()->assertNotNull(assemblerOptionsSpec.isTakeOwnership()),
+		  ()->assertNotNull(assemblerOptionsSpec.isValidateOnly()),
+		  ()->assertNotNull(assemblerOptionsSpec.getDefaultStyle()),
 		  ()->assertNotNull(assemblerOptionsSpec.getFirstBatesNumber()));	 
 	}
 	
```

Comment:
The reason you missed the typo above is because there are no tests that cover that branch of the code.  I'd like to see some more tests that cover the branches for each of properties of the AssemblerResult and that test that the results are correct.  The code coverage for target of this unit test class sits at about 68.5% which is OK, but not great.  You should typically aim for higher than 80% with the goal of being in the low-to-mid 90%s.